### ISSUE TITLE
VerticalStackPanelConstraints no longer works in iOS13.

### ISSUE THREAD

**Issue by MaxxDelusional** [FluentLayout/Cirrious.FluentLayout#57]
The constraints that are returned cause my content inside of the parent to no longer be visible.

**Comment by Gaisuru** [comment]
Are you using it inside a UIScrollView?

**Comment by MaxxDelusional** [comment]
Yes I am. The same code works fine in iOS12, but it breaks in iOS13.

**Comment by Gaisuru** [comment]
So, the issue is a change in iOS 13, not the library. iOS 13 now includes a indicator view as the first child in View.Subviews. So you can not just pass view.subviews anymore. You need to create a list of the views you are adding and then pass that in.

**Comment by Gaisuru** [comment]
We are working on updating the docs and examples to reflect this.

**Comment by MaxxDelusional** [comment]
Fair enough. I'll look forward to the updated docs.
